src/SFA.DAS.AssessorService.Web.Staff.Tests/Controllers/Roatp/AddRoatpOrganisationControllerTests.cs

```diff
@@ -121,6 +121,16 @@ public void Add_provider_type_model_without_legal_name_redirect_to_organisation_
             _client.VerifyAll();
         }
 
+        [Test]
+        public void Add_model_with_invalid_application_determined_date_redirect_to_xxx()
```

Comment:
Is this test name accurate, the expectation is for a view result, not a redirect?